android/app/src/main/java/org/uwaterloo/subletr/pages/listingdetails/ListingDetailsPageView.kt

```diff
@@ -131,8 +133,8 @@ fun ListingDetailsPageView(
 							contentDescription = stringResource(id = R.string.listing_image),
 							contentScale = ContentScale.FillWidth,
```

Comment:
`ContentScale` suggestions for the 250x250 container

* `ContentScale.Crop` center crop image and fit space
* `ContentScale.Fit` keep aspect ratio but have width or height whitespace for unfillable areas
* `ContentScale.FillBounds` fits space but lose aspect ratio

personally I like `ContentScale.Fit`

Reply:
Sounds good, but let's make that 256 x 256. Always best to stick with powers / multiples of 2 unless you have a good reason to not.

Reply:
another haiku 
multiples of two
sorry I was being faithful
to the figma mocks

Reply:
Figma mocks are hot garbage idk who made those...